frame/support/test/tests/pallet.rs

```diff
@@ -697,6 +705,13 @@ fn metadata() {
 					default: DecodeDifferent::Decoded(vec![0]),
 					documentation: DecodeDifferent::Decoded(vec![]),
 				},
+				StorageEntryMetadata {
+					name: DecodeDifferent::Decoded("Value2".to_string()),
```

Comment:
I think we might actually want to break the metadata here...

Maybe not right in this PR, but you can imagine someone will want to expose the name as "Balance" but use a custom prefix like "1".

As such, there would be a `name` property in the metadata, and a `prefix` property.

But this is something that would be much more annoying to merge in, and so probably should be an issue and we save it for when we already will break metadata.